docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+In another, Amazon calculated that a page load slowdown of just one second would cut conversions by
+7% -- costing them $1.6B in sales each year.
+footnote:[Source and more info: https://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales[fastcompany.com]]
+In short, a good {user-experience} keeps your users happy and improves your website's odds of success.
+
+[discrete]
+[[how-user-experience-works]]
+== How does {user-experience} work?
+
+// I'm not even sure if this section is necessary
+// Some of this is taken from the RUM docs
+
+{user-experience} metrics are powered by the {apm-rum-ref}[APM Real User Monitoring (RUM) agent].
+The RUM agent is installed as a dependency to your application,
+and it only takes a few lines of code to <<instrument-apps,get started>>.
+
+The RUM agent uses browser timing APIs, like https://w3c.github.io/navigation-timing/[Navigation Timing],
+https://w3c.github.io/resource-timing/[Resource Timing], https://w3c.github.io/paint-timing/[Paint Timing],
+and https://w3c.github.io/user-timing/[User Timing], to capture {user-experience}
+metrics every time a user hits one of your pages.
+This data is stored in {es}, where it can be visualized using {kib}.
+
+[discrete]
+[[user-experience-tab]]
+== {user-experience} in {kib}
+
+// Need some kind of app introduction here
+// Not quite sure yet...
+// ...any ideas?
+
+[discrete]
+[[user-experience-page-load]]
+=== Page load duration
+
+This high-level overview is your analysis starting point and answers questions like:
+How long is my server taking to respond to requests?
+How much time is spent parsing and painting that content?
+How many page views has my site received?
+
+You won't be able to fix any problems from viewing these metrics alone,
+but you'll get a sense of the big picture as you dive deeper into your data.
+
+[role="screenshot"]
+image::images/page-load-duration.png[User experience page load duration metrics]
+
+[discrete]
+[[user-experience-metrics]]
+=== {user-experience} metrics
+
+{user-experience} metrics help you understand the perceived performance of your website.
+For example, first contentful paint is the timestamp when the browser begins rendering content.
+In other words, it's around this time that a user first gets feedback that the page is loading.
+
+[role="screenshot"]
+image::images/user-exp-metrics.png[User experience metrics]
+
+// This is collapsed by default
+[%collapsible]
+.Metric reference
+====
+First contentful paint::
+Focusses on the initial rendering and measures the time from when the page starts loading to when
+any part of the page's content is displayed on the screen.
+The agent uses the https://www.w3.org/TR/paint-timing/#first-contentful-paint[Paint timing API] available
+in the browser to capture the timing information.
+FCP is captured as transaction mark for `page-load` transaction for all chromium-based browsers
```

Comment:
I'd take out the part that says this is only in chromium based browsers as Opera supports it and Safari are about to release support as well (it's in their tech preview).

Reply:
Yup, this makes sense.